doc/manual/manual.md

```diff
@@ -2892,6 +2896,68 @@ point of a call to `f`. This specification also constrains `y` to prevent
 signed integer overflow resulting from the `x + y` expression in `f`,
 which is undefined behavior in C.
 
+### MIR static items
```

Comment:
The documentation for `mir_static` and `mir_static_initializer`. There is more that could be said here about how mutable static items interact with overrides, but I need to implement overrides first before I can meaningfully talk about this :)